FeedAPIChallenge/Feed API/FeedImagesMapper.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  Copyright © 2018 Essential Developer. All rights reserved.
+//
+
+import Foundation
+
+enum FeedImagesMapper {
+	private static let validStatus = 200
+
+	private struct FeedImageRemoteList: Decodable {
+		let items: [FeedImageRemoteItem]
+
+		var feedImageList: [FeedImage] {
+			items.map { $0.feedImage }
+		}
+	}
+
+	private struct FeedImageRemoteItem: Decodable {
+		let imageId: UUID
+		let imageDesciption: String?
+		let imageLocation: String?
+		let imageUrl: URL
+
+		enum CodingKeys: String, CodingKey {
+			case imageId = "image_id"
+			case imageDesciption = "image_desc"
+			case imageLocation = "image_loc"
+			case imageUrl = "image_url"
+		}
+
+		var feedImage: FeedImage {
+			.init(id: imageId, description: imageDesciption, location: imageLocation, url: imageUrl)
+		}
+	}
+
+	static func handleSuccessCase(with data: Data, from response: HTTPURLResponse) -> FeedLoader.Result {
```

Comment:
Suggestion:
This method name can be confusing. A word "handle" usually implies some side effects but this method has no side effects and instead returns a value. So in terms of CQS principle it is a query, not a command. 
Another thing that can confuse is that when this function is called it is written like "hey mapper! handle this success case, please" which a little bit contradicts with the name of the type. Mapper is expected to "map" but not to "handle". Mappers gonna map map map 😅. Thus it is better to call this function simply `map` 
```suggestion
	static func map(data: Data, from response: HTTPURLResponse) -> FeedLoader.Result {
```